src/datahike/constants.cljc

```diff
@@ -5,3 +5,4 @@
 (def ^:const emax 0x7FFFFFFF)
 (def ^:const txmax 0x7FFFFFFF)
 (def ^:const implicit-schema {:db/ident {:db/unique :db.unique/identity}})
+(def ^:const max-db-caches 5)
```

Comment:
Shouldn't this be a configurable parameter?

Reply:
Yes - maybe by environment variable? 
It can't be given with the database config since it is a global parameter.

Reply:
Could we do it on a database id level (not value, but uuid that is globally fixed for each db)? I think @kordano has not added that yet, but in the longer run it might be good to be able to decide how big the cache should be for individual databases.

Reply:
That would be a good idea. Would make it more intuitive than defining it only per database snapshot, which is how it works at the moment